docker/base.yaml

```diff
@@ -18,9 +18,6 @@ version: "3.4"
 services:
   base_container:
     image: pdo_base:${PDO_VERSION:-latest}
-    build:
-      context: .
-      dockerfile: pdo_base.dockerfile
-      args:
-        ADD_APT_PKGS: ${ADD_APT_PKGS}
-        PDO_VERSION: ${PDO_VERSION:-latest}
+    # build containers only via Makefile and directly via docker build
+    # as we use containers also outside of docker-compose to prevent
+    # double-build
```

Comment:
ah... way ahead of me! see my comment about add SGX_MODE to the versions & i think this will help a LOT.

Reply:
see above, i don't think adding sgx-mode to tag is the right thing to do